R/check-cardinalities.R

```diff
@@ -46,6 +46,8 @@
 #'
 #' @name check_cardinality
 #'
+#' @return Functions invisibly return `TRUE`, if the check is passed. Otherwise an error is thrown and the reason for it is explained.
```

Comment:
Wording, here and everywhere:

```suggestion
#' @return Functions return `TRUE`, invisibly, if the check is passed. Otherwise an error is thrown and the reason for it is explained.
```

Is there a reason we don't return the first argument here?

Reply:
Can't really remember, but maybe the thought was that we are checking if a certain cardinality is given, and `TRUE` seems like an appropriate return value in case it is fulfilled.

But yeah, let's change it?